### PR TITLE
#153 added logo to pages and README

### PR DESCRIPTION
**Pull request by krzyk**
#153

I've added information about logo in a separate section of the README.md. Logo has been added to the page header (and centered horizontally). I've also resized the original logo to 50% of the original size, as the large one was just to big for the page.

### CODE REVIEW

**Comment by karato** [comment]
Many thanks, I will find someone to review it before we merge

**Comment by karato** [comment]
@alevohin do this review

**Comment by alevohin** [comment]
@krzyk readmt.md contains link to obsoleted logo https://camo.githubusercontent.com/e9f70fcdcc4f9ea49cf91e36ff8b75447d0aa683/687474703a2f2f7777772e6e65726f6465736b2e636f6d2f6c6f676f2e706e67
Let's replace it too.

File: src/main/java/com/nerodesk/takes/TsApp.java [hunk 1]
```diff
@@ -148,6 +148,7 @@ public void run() {
                     "text/css"
                 )
             ),
+            new FkRegex("/images/.*", new TsClasspath()),
```
Comment (alevohin) [inline src/main/java/com/nerodesk/takes/TsApp.java]: Lets add unit test for it.

**Comment by alevohin** [comment]
@krzyk thank you! Please see comment above.

**Comment by krzyk** [comment]
@alevohin please check now

**Comment by alevohin** [comment]
@krzyk thank you!

**Comment by alevohin** [comment]
@rultor merge

File: src/main/java/com/nerodesk/takes/TsApp.java (hunk 1 above)
Comment (yegor256) [inline src/main/java/com/nerodesk/takes/TsApp.java]: the image is returned without any Content-Type set. some browsers may complain. let's wrap this into `TsWithType`

**Comment by yegor256** [comment]
@krzyk one comment from me

**Comment by krzyk** [comment]
@yegor256 updated, please check

**Comment by ghost** [comment]
@yegor256 let's confirm this.

**Comment by yegor256** [comment]
@rultor merge this

**Comment by ghost** [comment]
@krzyk thanks.

**Comment by karato** [comment]
@alevohin Much obliged! I have added **19 mins** to your account in payment "54979624", 32 hours and 7 mins spent; review comments (c=4) added as a bonus; +19 added to your rating, at the moment it is: [+2145](http://www.netbout.com/b/35105?open=rating)

**Comment by karato** [comment]
@rultor deploy

**Pull request merged**